src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -1129,13 +1110,13 @@ internal bool HasInvariantCultureName
         /// Helper function overloads of GetCachedReadOnlyCulture.  If lcid is 0, we use the name.
         /// If lcid is -1, use the altName and create one of those special SQL cultures.
         /// </summary>
-        internal static CultureInfo GetCultureInfoHelper(int lcid, string name, string altName)
+        internal static CultureInfo? GetCultureInfoHelper(int lcid, string? name, string? altName)
         {
             // retval is our return value.
             CultureInfo retval;
```

Comment:
Should retval be set as nullable? to match the return type and also since we're doing retval != null in various places.